pom.xml

```diff
@@ -543,6 +548,12 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.8.0</version>
+                <configuration>
+                    <compilerArgs>
+                        <arg>-Xlint:deprecation</arg>
+                        <arg>-Xlint:unchecked</arg>
+                    </compilerArgs>
+                </configuration>
```

Comment:
what about maven.compiler.showWarnings (property) or showWarnings (configuration item)? Is it set, too?

Reply:
done